src/data_internal.h

```diff
@@ -24,7 +24,7 @@ message_type_string(message_type type)
       case MESSAGE_TYPE_DELETE:
          return "delete";
       default:
-         debug_assert(FALSE, "Invalid message type");
+         debug_assert(FALSE, "Invalid message type=%d", type);
          return "invalid";
    }
```

Comment:
Maybe add a specific `MESSAGE_TYPE_INVALID` case?

Reply:
@ajhconway - I added a case for the message type `MESSAGE_TYPE_INVALID` and for the newly added message type `MESSAGE_TYPE_PIVOT_DATA`.